src/css/header.scss

```diff
@@ -0,0 +1,30 @@
+@import './core/constants';
+@import './core/mixins';
+
+$hearder_height: 60px;
```

Comment:
Need constant naming convention here, stick to 1 style `-` or `_`.
We prefer `$headerHeight`